src/main/java/org/reactome/release/downloaddirectory/ProtegeExporter.java

```diff
@@ -123,8 +124,8 @@ private void createProtegeFiles(MySQLAdaptor dba)
 				// know they could still be running.
 				Runtime.getRuntime().addShutdownHook(new Thread( () ->
 				{
-					logger.info("Shutting down thread pool. IF this program was terminated prematurely, there might still be orphan Perl processe runing."
-								+ " Check to see if any are still running, and kill them if necssary.");
+					logger.info("Shutting down thread pool. IF this program was terminated prematurely, there might still be orphan Perl processes running."
```

Comment:
Should this be phrased as a WARN? Maybe it makes sense to have the 'shutting down thread pool' as INFO and the 'IF' statement as a WARN?

Reply:
@SolomonShorser-OICR What do you think about separating this logging statement into one INFO and one WARN?  Also, is there a way to catch a program interruption and shutdown any orphan processes?

Reply:
It's _probably_ possible to detect and kill orphan Java processes started by Runtime. I'm really _not_ sure how *easy* it would be. How often does this problem happen? How much effort is available to be invested? ;)

Reply:
Ok, I guess just split to two logging statements, as per @cookersjs's suggestion.

Reply:
Okay, done: https://github.com/reactome/release-download-directory/pull/4/commits/f0688b189ef96fb9b692bc5b6142f90bd2cd241a